tests/Forms/DiffFieldTest.php

```diff
@@ -45,59 +45,53 @@ public function testEscaping(
         string $oldValue,
         string $newValue,
         string $expected,
-        string $message
     ) {
         // get $emptyPlaceholder here instead of provideEscaping to prevent
         // BadMethodCallException: No injector manifests available
         // error in dataProvider method
-        $emptyPlaceholder = ReadonlyField::create('na')->Value();
+        $emptyPlaceholder = ReadonlyField::create('na')->getFormattedValue();
         $emptyPlaceholderNoTags = strip_tags($emptyPlaceholder);
         $expected = str_replace('$emptyPlaceholderNoTags', $emptyPlaceholderNoTags, $expected);
         $expected = str_replace('$emptyPlaceholder', $emptyPlaceholder, $expected);
-        $newField = new $className('Test', 'Test', $oldValue);
+        $newField = new $className('Test', 'Test', $newValue);
         $diffField = DiffField::create('DiffTest');
         $diffField->setComparisonField($newField);
-        $diffField->setValue($newValue);
-        $this->assertSame($expected, $diffField->Value(), $message);
+        $diffField->setValue($oldValue);
+        $this->assertSame($expected, $diffField->getFormattedValue());
     }
 
     public static function provideEscaping()
     {
         return [
-            [
-                ReadonlyField::class,
-                'Something',
-                'Something <strong>bold</strong>',
-                'Something <del>&lt;strong&gt; bold &lt;/strong&gt;</del>',
-                'Non HTML field is escaped'
+            'readonly-add-bold' => [
+                'className' => ReadonlyField::class,
+                'oldValue' => 'Something',
+                'newValue' => 'Something <strong>bold</strong>',
+                'expected' => 'Something <ins>&lt;strong&gt; bold &lt;/strong&gt;</ins>',
```

Comment:
Why did the expected value of this test case change? Haven't looked at the others, I assume something has gone awry.

Reply:
The test was literally back to front where it was testing things as if the new version was the old version and vice versa.  Note that in the test itself (along with the dataProvider) I've reversed things. I fixed it as unrelated refactoring.